tests/app/service/test_rest.py

```diff
@@ -371,41 +371,41 @@ def test_create_service_should_throw_duplicate_key_constraint_for_existing_email
             assert "Duplicate service name '{}'".format(service_name) in json_resp['message']['name']
 
 
-def test_update_service(notify_api, notify_db, sample_service):
+def test_update_service(client, notify_db, sample_service):
```

Comment:
Do we also need to update the `create_service` test to check that it creates with the default `DVLA_ORG_HM_GOVERNMENT`?